assets/libraries/k8s/library.rego

```diff
@@ -16,6 +16,18 @@ checkKind(currentKind, listKinds) {
 	currentKind == listKinds[i]
 }
 
+CheckIFPdbExists(statefulset) = result {
+	pdb := input.document[j]
+	pdb.kind == "PodDisruptionBudget"
+	result := containsLabel(pdb, statefulset.spec.selector.matchLabels)
+} else = false {
+	true
+}
```

Comment:
This function should not be in the generic libraries as it's too specific for a query. Also please change the name to a more appropriate name like "isPodDisruptionBudget"